theme/layouts/default.ejs

```diff
@@ -6,10 +6,11 @@
   <body>
     <header>
       <div>
-        <h1><%= page.header %></h1>
-      </div>
-    </header>
-    <% include _navigation %>
+        <h1><a href="#"><%= page.header %></a></h1>
```

Comment:
This's an interesting example of making a decision about semantic elements.

- `CBA Chromium Browser Automation (extension for chrome browser automation)` is the main title of the page
```
<h1><a><%= page.header %></a> <%= page.slogan %></h1>
```

- `CBA` is the main title of the page and `Chromium Browser Automation (extension for chrome browser automation)` is a paragraph text there.
```
<h1><a><%= page.header %></a></h1>
<p><%= page.slogan %></p>
```

- - `CBA` is the main title of the page and `Chromium Browser Automation (extension for chrome browser automation)` is the subtitle of the page.
```
<h1><a><%= page.header %></a></h1>
<h2><%= page.slogan %></h2>
```

It's tricky to make a choice here, but let's go with second suggestion here, unless you have a preference to other options and we can revisit that later if we decide it can have a SEO implications:
```
<h1><a><%= page.header %></a></h1>
<p><%= page.slogan %></p>
```

Anyway having `h3` after `h1` here is not what we need I guess.

Reply:
OK. Done